src/promptflow-evals/tests/evals/e2etests/test_evaluate.py

```diff
@@ -80,8 +80,8 @@ def _get_run_from_run_history(flow_run_id, ml_client, project_scope):
 
 
 @pytest.mark.usefixtures("recording_injection")
-@pytest.mark.e2etest
 class TestEvaluate:
+    @pytest.mark.localtest
```

Comment:
Same here. Apply @pytest.mark.localtest to the class level.

Reply:
The marks are not mutually exclusve, in this case `pytest -m localtest ... ` will also run azure tests.

Reply:
Fixed.